cookbooks/fb_postfix/attributes/default.rb

```diff
@@ -68,7 +78,7 @@
     'sendmail_path' => '/usr/sbin/sendmail.postfix',
     'setgid_group' => 'postdrop',
     'manpage_directory' => '/usr/share/man',
-    'mailq_path' => '/usr/bin/mailq.postfix',
+    'mailq_path' => mailq_path,
```

Comment:
This changes the default path on non-Debian platforms, is that intentional?

Reply:
Oops, the conditional above is supposed to be the other way around. Will fix. Nice catch.